compiler/main/driver.cpp

```diff
@@ -516,19 +516,19 @@ static void setupChplHome(const char* argv0) {
     // E.g. /usr/lib/chapel/1.16/runtime/lib
     rc = snprintf(CHPL_RUNTIME_LIB, FILENAME_MAX, "%s/%s/%s/%s",
                   get_configured_prefix(), // e.g. /usr
-                  "/lib/chapel",
+                  "lib/chapel",
```

Comment:
Do you know that `get_configured_prefix` always ends in a `/` if it is not empty?

Reply:
The format string has the slashes between components